CHANGELOG.adoc

```diff
@@ -15,6 +15,42 @@ https://github.com/oxidecomputer/dropshot/compare/v0.4.0\...HEAD[Full list of co
 // configured in release.toml.  DO NOT change the format of the headers or the
 // list of raw commits.
 
+`HttpServer` has been split into two objects, `HttpServerStarter`, and
```

Comment:
I think there are some asciidoc formatting nits here.  (The list doesn't render as a list, at least on GitHub.)

Reply:
Thanks for the heads up, looks like this is fixed now.